web/src/features/moreCast2/components/TabbedDataGrid.tsx

```diff
@@ -453,19 +456,61 @@ const TabbedDataGrid = ({ morecast2Rows, fromTo, setFromTo }: TabbedDataGridProp
     setShowHideColumnsModel(newModel)
   }
 
+  /**
+   * Reset forecast rows to their default state. Temp, RH, Wind Dir & Speed are cleared,
+   * Precip is set to 0, and GC is carried forward from last submitted value.
+   */
+  const resetForecastRows = () => {
```

Comment:
I think this function is good but `TabbedDataGrid` is getting huge, maybe we can move it to the reset button component where it takes the setAllRows, fillGrassCuringForecast as props to call back up to parent.